pkg/syncer/metadatasyncer.go

```diff
@@ -261,6 +271,18 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 				}
 				metadataSyncer.volumeManagers[vcconfig.Host] = volumes.GetManager(ctx, vCenter, nil, false)
 			}
+			// Add informer on CSINodeTopology instances and update metadataSyncer.topologyVCMap parameter.
+			nodeMgr = node.GetManager(ctx)
+			k8sConfig, err := k8s.GetKubeConfig(ctx)
+			if err != nil {
+				return logger.LogNewErrorf(log, "failed to get kubeconfig with error: %v", err)
+			}
+			metadataSyncer.topologyVCMap = make(map[string]map[string]struct{})
+			err = startTopologyCRInformer(ctx, k8sConfig)
```

Comment:
Do you need any RBAC rules for this new informer?

Reply:
Not required, syncer already contains the CSINodeTopology reconciler which gets and watches the instances.